back/engines/commercial/content_builder/app/services/content_builder/layout_service.rb

```diff
@@ -11,5 +11,19 @@ def select_craftjs_elements_for_type(craftjs, type)
     def craftjs_element_of_type?(elt, type)
       elt.is_a?(Hash) && elt['type'].is_a?(Hash) && elt.dig('type', 'resolvedName') == type
     end
+
+    def images(layout)
```

Comment:
> This will probably become a method at the model level (or something like this), that returns a collection of all layout images currently associated with the layout.

> This is likely to be useful in many places, including [CL-2753 BE Job to clean up layout images not associated with any layout](https://citizenlab.atlassian.net/browse/CL-2753)

https://github.com/CitizenLabDotCo/citizenlab/pull/3850#discussion_r1094434808

Reply:
> This is likely to be useful in many places, including

Until it's the case, I wouldn't put it in the real code. Because if it's not used, nobody will ever remove it.

Reply:
OK, removed.